### ISSUE TITLE
Работает на половину

### ISSUE THREAD

**Issue by Varezh45** [dovecoteescapee/ByeDPIAndroid#176]
### Describe the bug / Описание ошибки

Работает отлично, но видео загружает по пол минуты, с перемоткой тоже самое. Ещё может видео загружаться, но намертво зависать через 10 секунд.

### To Reproduce / Как воспроизвести

1: Зайти в Ютуб
2: Открыть видео/шортс

### Expected behavior / Ожидаемое поведение

Шоби видево загружалас

### Environment / Окружение

**Smartphone / Смартфон:**
- Device: [Xiaomi Redmi 11S]
- Android version: [e.g. 11]
- ByeDPI version: [e.g. 1.0.2]
- МГТС домашний

### Before you submit / Прежде чем отправить

- [X] I have searched the open and closed issues for duplicates / Я искал(а) открытые и закрытые задачи на предмет дубликатов
- [ ] This is not a question, feature request, or general feedback / Это не вопрос, предложение новой функциональности или общий отзыв

**Comment by yunon8f** [comment]
вообще ни на каких конфигах ютуб не хочет работать

**Comment by K1llbi** [comment]
> вообще ни на каких конфигах ютуб не хочет работать

согласен, на андроиде 7, версия 1.2.0 работает не на полную, а вам советую обновиться до версии 1.2.0, на вашей модели более менее свежий андроид, попробуйте out-of..., split 2, oob data g, udp fake 5, на андроид 14 работает

**Comment by Alexoff442** [comment]
На телеке не работал, пока не поставил настройки такие же как на телефоне, хотя на телефоне всё стояло по дефолту

**Comment by Alexoff442** [comment]
Сейчас вроде работает, тормоза не наблюдаю

**Comment by Pareq** [comment]
У меня все так же, только телефон другой

**Comment by Muzuvaka** [comment]
++

**Comment by rellikmil** [comment]
Мне помог -n “www.google.com” -q 1+s -O 1 -s 25+s -t 5, предзагрузка видео сократилась в общем, так скажем, до этого стабильно 10-15 секунд на каждом видео

**Comment by Nior21** [comment]
> Мне помог -n “www.google.com” -q 1+s -O 1 -s 25+s -t 5, предзагрузка видео сократилась в общем, так скажем, до этого стабильно 10-15 секунд на каждом видео
Эти настройки сработали для меня в подмосковье, на Бейрель. Без этих параметров перестало работать уже неделю как. Спасибо за ваш труд!

**Comment by roundedrectangle** [comment]
для меня заработало следующее (на обычном byedpi, но думаю на android тоже сработает):
- либо использовать [этот piped](https://watch.leptons.xyz/)
- либо использовать [этот invidious](https://inv.nadeko.net/) - стабильнее, также с ним рекомендую использовать [свои временные данные](https://git.nadeko.net/Fijxu/invidious/wiki/Generate-a-po_token-and-visitor_data-for-inv.nadeko.net), вход в аккаунт в них не нужен

**Comment by roundedrectangle** [comment]
и ещё, это не совсем баг, такие вопросы лучше спрашивать в [обсуждениях](https://github.com/dovecoteescapee/ByeDPIAndroid/discussions)

**Comment by sanrit24** [comment]
более менее у меня с такими работает -q1+s -s25+s -o5+s f-1 -S -As

**Comment by Forsar190** [comment]
Здравствуйте сомо приложение работает Но почему то когда оно работает не работает уже телеграмм 

Я не видел не где чтобы об этом где то хоть раз говорили по этому написал сюда 

Можно ли в будущих обновлениях добавить белый список приложений или сайтов на каторых оно не будет влиять так называемое туннелирования?
